### PR TITLE
Fix typo in teacherspet.jar in UG

### LINKED ISSUE #241: [PE-D][Tester C] jar file is wrongly named in UG
Quick Start section, name of jar file should be `teachersPet.jar`

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: neosunhan/ped#1

### PR DESCRIPTION
**Pull request by gnahzeus**
Closes #241

### CODE REVIEW

**Review by seanflyyy — APPROVED** [review]
LGTM! Not much changed.

**Review by sjoann — APPROVED** [review]
LGTM !

File: docs/UserGuide.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ For a full list of commands and detailed instructions, head to the [Features](#f
 ## Quick start
 
 1. Ensure you have Java `11` or above installed in your Computer.
-2. Download the latest `teacherpet.jar` from https://github.com/AY2223S1-CS2103T-T09-4/tp/releases when made available.
+2. Download the latest `teacherspet.jar` from https://github.com/AY2223S1-CS2103T-T09-4/tp/releases when made available.
```
Comment (Echomo-Xinyu) [inline docs/UserGuide.md:57]: I believe this is named `teachersPet.jar` on GitHub right?

Comment (gnahzeus) [reply docs/UserGuide.md:57]: Excellent catch Xinyu! Thanks for the input 👍

**Pull request merged**